wordpress-mobile/WordPress-iOS
# Add Jetpack powered banner to Reader Search

Fixes #NA

This PR adds a "Jetpack powered" banner in the reader search of the WordPress app (see screenshots).
The banner won't be visible on iPhone landscape.
This PR also removes (temporarily) the "Search WordPress for a site or a post" label that appeared below the "Clear search history" button.

Newer iPhone portrait | iPhone landscape | older iPhone portrait with larger fonts
-- | -- | --
<img width=300 src=https://user-images.githubusercontent.com/34376330/179321205-c979a10c-2c36-4287-93b3-84007067c252.png> | <img width="589"  src="https://user-images.githubusercontent.com/34376330/179321447-6076e13d-0b3b-49e0-bf27-05d39ea59172.png"> | <img width=300 src=https://user-images.githubusercontent.com/34376330/179321215-5300613f-d1e7-4345-ad5c-745df95a272f.png>


To test:
- build/run the `WordPress` target from this branch
- Go to Reder and select search in the top-right corner
- on iPad and iPhone portrait: make sure that the "Jetpack powered" banner appears
- Rotating to landscape on iPhone (or starting the app in landscape) make sure that the banner does not appear
- Stop the execution and build/run the `Jetpack` target from this branch
- Repeat the above steps and make sure that the "Jetpack powered" banner does not show in any step.

## Regression Notes
1. Potential unintended areas of impact
Low, it adds an `inputAccessoryView` to the `serchBar` in `ReaderSearchViewController`
This PR also adjusts heights calculation in `ReaderSearchSuggestionsViewController` and removes a label in `ReaderSearchViewController`

2. What I did to test those areas of impact (or what existing automated tests I relied on)
test on several devices and font sizes

3. What automated tests I added (or what prevented me from doing so)
none
PR submission checklist:
